indico/modules/rb/templates/emails/reservations/creation_email_to_user.txt

```diff
@@ -1,31 +1,78 @@
 {% extends 'rb/emails/reservations/base_email_to_user.txt' %}
 {% set excluded_days = reservation.find_excluded_days().count() %}
+{%- set prebooking = not reservation.is_accepted -%}
+{% block subject_prefix %}{% endblock %}
+{% block subject -%}
+    {% if prebooking -%}
+        {% trans room=reservation.room.full_name, start=reservation.start_dt|format_datetime('dd/MM/yyyy HH:mm', locale='en_GB') -%}
+            [{{ room }}] Pre-Booking awaiting acceptance ({{ start }})
+        {%- endtrans %}
+    {%- else -%}
+        {% trans room=reservation.room.full_name, start=reservation.start_dt|format_datetime('dd/MM/yyyy HH:mm', locale='en_GB') -%}
+            [{{ room }}] Booking confirmed ({{ start }})
+        {%- endtrans %}
+    {%- endif %}
+{%- endblock %}
 
-{% block body -%}
-{%- block booking_details -%}
-The conference room {{ reservation.room.full_name }}
-has been {% block prebooked_prefix %}{% endblock %}booked for {{ reservation.booked_for_name }}
-{% if reservation.repeat_frequency == RepeatFrequency.NEVER -%}
-    on {{ reservation.start_dt | format_date('EEEE dd/MM/yyyy', locale='en_GB') }} from {{ reservation.start_dt | format_time(locale='en_GB') }} to {{ reservation.end_dt | format_time(locale='en_GB') }}.
-{%- else -%}
-    {{ RepeatMapping.get_message(reservation.repeat_frequency, reservation.repeat_interval) }} from {{ reservation.start_dt | format_date('EEEE dd/MM/yyyy', locale='en_GB') }} to {{ reservation.end_dt | format_date('EEEE dd/MM/yyyy', locale='en_GB') }} between {{ reservation.start_dt | format_time(locale='en_GB') }} and {{ reservation.end_dt | format_time(locale='en_GB') }}.
-{%- endif %}
-Reason: {{ reservation.booking_reason }}
-{%- endblock -%}
-{%- block confirmed_booking -%}
-{%- if excluded_days %}
-    (Note that there are {{ excluded_days }} excluded days. For further info, check your reservation)
-{%- endif %}
+{% block email_body -%}
+    {%- filter dedent -%}
+        {%- block booking_details -%}
+            {% if prebooking -%}
+                {% if reservation.repeat_frequency == RepeatFrequency.NEVER -%}
+                    {% trans notrimmed room=reservation.room.full_name, name=reservation.booked_for_name, date=reservation.start_dt | format_date('EEEE dd/MM/yyyy', locale='en_GB'), from=reservation.start_dt | format_time(locale='en_GB'), to=reservation.end_dt | format_time(locale='en_GB') -%}
```

Comment:
- Can we wrap this over multiple lines? 🙈 
- Removing the whitespace around the `|` filter operator makes this more readable (the grouping feels really weird when reading it right now)
- I don't think forcing fixed date formats and a locale is a good idea here (we don't want dd/mm/yyyy format in en_US for example, and french speakers probably don't want english weekday names ;)). Try `|format_skeleton('EEEEddMMyyyy')` instead, this is smart enough to pick the closest format for the current locale that matches the sleketon:

```python
In [11]: format_skeleton(dt(2022, 9, 11), 'EEEEddMMyyyy', 'en_GB', 'Europe/Zurich')
Out[11]: 'Sunday, 11/09/2022'

In [12]: format_skeleton(dt(2022, 9, 11), 'EEEEddMMyyyy', 'de_DE', 'Europe/Zurich')
Out[12]: 'Sonntag, 11.09.2022'

In [13]: format_skeleton(dt(2022, 9, 11), 'EEEEddMMyyyy', 'fr_FR', 'Europe/Zurich')
Out[13]: 'dimanche 11/09/2022'

In [14]: format_skeleton(dt(2022, 9, 11), 'EEEEddMMyyyy', 'en_US', 'Europe/Zurich')
Out[14]: 'Sunday, 09/11/2022'
```